UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API.UnitTests/Controllers/SubscriptionControllerTests.cs

```diff
@@ -69,12 +73,29 @@ public async Task WhenPostInvalidNullInputParameters_ThenReceiveBadRequest()
         }
 
         [Test]
-        public async Task WhenPostValidPayload_ThenReceiveSuccessfulResponse()
+        public async Task WhenPostInvalidNotificationTypesInPayload_ThenRecieveBadRequest()
         {
```

Comment:
Please correct spelling as WhenPostInvalidNotificationTypeInPayload_ThenReceiveBadRequest

Reply:
Thanks for highlighting this, done required changes